src/setting.ts

```diff
@@ -30,6 +30,9 @@ export class KeyValue<T, S> {
 
 export class CustomSettings {
   public ignoreUploadFiles: string[] = [

```

Comment:
can you add `.DS_Store` file also here. Because this file is changed for mac only users and auto upload gets into the loop.
Please include lock file also in this list.

Reply:
Sure